WalletWasabi.Gui/Controls/WalletExplorer/SendTabBaseView.xaml.cs

```diff
@@ -0,0 +1,19 @@
+using Avalonia;
+using Avalonia.Controls;
+using Avalonia.Markup.Xaml;
+
+namespace WalletWasabi.Gui.Controls.WalletExplorer
+{
+	public class SendTabBaseView : UserControl
```

Comment:
This is used as a control, not a base class: rename to SendControlView.

Reply:
Suggest `SendControl`